### DISCUSSION TITLE
iOS limitations

### DISCUSSION THREAD

**Discussion by gianpietro1** [gilbarbara/react-spotify-web-playback#135] [General]
Hi, the [SDK documentation](https://developer.spotify.com/documentation/web-playback-sdk/#supported-browsers) says "The SDK does not work when the browser is running in the background.", and indeed, when using an iOS browser (Safari, Chrome,...) audio stops when track changes.
Anyone knows any workaround for this?
Thanks.

**Comment by gilbarbara** [comment]
If the browser tab is in the background, there's no workaround. Only full mobile apps can have that.
